docs/proposals/network/README.md

```diff
@@ -0,0 +1,66 @@
+# Software Defined Network
+
+## Analysis
+
+### Examples
+
+- Canal
+- Calico
+- Flannel
+- Weave
+- Cilium
+
+![benchmark-cni](https://miro.medium.com/v2/resize:fit:1400/1*zDiFQZVXuL923x0LbmXMaA.png)
+
+### Sources
+
+- [CNI](https://github.com/containernetworking/cni) also makes a good framework for creating a new container networking project from scratch
+- [Networks - treeptik k8s](https://treeptik.gitbook.io/k8s/fundamentals/)network
+- [Explain NS linux](https://www.youtube.com/watch?v=j_UUnlVC2Ss)
+- [Spec - CNI](https://github.com/containernetworking/cni/blob/main/SPEC.md)
+
+<br>
+
+## Network domain
+
+### Definition
+
+The goal is to be able to:
+
+- manage networks through the software
+- isolate instance/pods with isolated networks
+- apply network security policies
+
+Options:
+
+- a library to manage networks directly from the Node Agent
+- a deamon running on each worker nodes and another on the master node
+
```

Comment:
A third option is a binary to be called directly by the agent. This is the CNI approach.

Reply:
Previously, we didn't understood how the CNI approach worked, but now we think it's more comprehensible

You can see our approach in the last commits:

- for the [definition](https://github.com/dev-sys-do/orka/blob/8e83b9971ca8364c6bd3aeec86938dca99cfbc0b/docs/proposals/network/README.md?plain=1#L26)
- and the [plugin implementation](https://github.com/dev-sys-do/orka/blob/8e83b9971ca8364c6bd3aeec86938dca99cfbc0b/docs/proposals/network/README.md?plain=1#L74)